pkg/kubectl/delete.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2014 The Kubernetes Authors.
+Copyright 2018 The Kubernetes Authors.
```

Comment:
This GitHub diff is misleading. What I actually did:
- Renamed `delete.go` -> `reapers.go` to better reflect the code living in that file (hopefully soon to be deleted)
- Created a new `delete.go` file with utility functions for deletion.